src/crewai/crew.py

```diff
@@ -154,6 +155,10 @@ class Crew(BaseModel):
         default=False,
         description="Plan the crew execution and add the plan to the crew.",
     )
+    planning_llm: Optional[Any] = Field(
```

Comment:
Shouldn't we set this to default None here? 

The reason I say that is because when a crew is created, it will require a openai key going forward even if the user doesn't want to do planning or use open ai.

Reply:
If I set it to None, it will force None as a parameter when we pass it to the planning function. 

Good catch, I'll change it so it forces the opener as the llm after the planning = True is verified